salt/edx/mitx_etl.sls

```diff
@@ -49,9 +49,9 @@ ensure_upload_bucket_exists:
 
 add_task_to_cron:
   cron.present:
-    - name: 'source /mitx/mitx_etl/bin/activate && python3 /mitx/mitx_residential_etl.py'
+    - name: 'source /mitx/mitx_etl/bin/python3 /mitx/mitx_residential_etl.py'
```

Comment:
You don't need the `source` command at all since you are providing the full path to the python3 binary.